api/message.go

```diff
@@ -130,6 +130,10 @@ func (a *MessageAPI) CreateMessage(ctx *gin.Context) {
 	message := model.Message{}
 	if err := ctx.Bind(&message); err == nil {
 		message.ApplicationID = a.DB.GetApplicationByToken(auth.GetTokenID(ctx)).ID
+		// If no title was specified always use the application name
+		if len(message.Title) == 0 {
```

Comment:
Just comparing the strings looks a little cleaner IMO.
```suggestion
		if message.Title == "" {
```